src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+
+import { BackButton } from '../../common/nav-button';
+import Button from '../../../components/button';
+import BoxDrive140 from '../../../illustration/BoxDrive140';
+
+import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
+import StaticVersionsItem from './StaticVersionsItem';
+
+import './StaticVersionsSidebar.scss';
+
+type Props = {
+    isLoading: boolean,
+    onUpgradeClick?: () => void,
+    parentName: string,
+    showUpsellWithPicture: boolean,
+};
+
+const StaticVersionsSidebar = ({ onUpgradeClick, isLoading, parentName, showUpsellWithPicture }: Props) => {
+    return (
+        <div
+            className="bcs-static-Versions"
+            role="tabpanel"
+            data-resin-component="preview"
+            data-resin-feature="versions"
+        >
+            <div className="bcs-static-content-header">
+                <h3 className="bcs-static-title">
+                    <>
+                        <BackButton data-resin-target="back" to={`/${parentName}`} />
+                        <FormattedMessage {...messages.versionsTitle} />
+                    </>
+                </h3>
+            </div>
+            <div className="bcs-static-scroll-content-wrapper">
+                <LoadingIndicatorWrapper
+                    className="bcs-static-Versions-content"
+                    crawlerPosition="top"
+                    isLoading={isLoading}
+                >
+                    <section>
+                        <h4 className="bcs-static-VersionsGroup-heading">
+                            <FormattedMessage {...messages.versionsToday} />
+                        </h4>
+                        <ul>
+                            <li>
```

Comment:
Matter of taste at this size (3 items), but I'd loop over `[1, 2, 3]` and assemble it dynamically to avoid having to maintain 3 entries.

Reply:
omg, tbh I am really proud that I changed to doing this before reading your comment lol